yucca/planning/dataset_properties.py

```diff
@@ -59,21 +65,24 @@ def create_properties_pkl(data_dir, save_dir, suffix=".nii.gz"):
                 completed += 20
                 print(f"Property file creation progress: {completed}%")
                 sys.stdout.flush()
-            image = nib.load(subject)
+            image = read_any_file(subject)
```

Comment:
It's a function that will read niftis, numpys, csv's, txts etc and return them as a nifti if possible otherwise a np array. Its purpose is basically just to read any file format we could expect and return it in a format we can work with

Reply:
I really do not like this tbh. I think it is *very* important to be able to know whether I have a nifti, a np.array and whether the code will explode in my head by just reading the naming of it. This can return so many things, and then I have to do check on the return value to know what I can do with. To me, this is the high way to spaghettiland.

Reply:
Yes (we figured it out in person, in a boxing ring) and I'll rename multiple functions to improve readability